_posts/2021-11-10-release-0.9.13.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+comments: true
+title:  "CARLA 0.9.13 Release"
+subtitle: "User defined traffic manager paths, carla-agents Python package, user defined pedestrian animations and new vehicle behaviours"
+description: "User defined traffic manager paths, carla-agents Python package, user defined pedestrian animations and new vehicle behaviours"
+author: "@MattRowe18"
+image: 'img/carla.jpg'
+background: '/img/posts/2021-08-02/town10.png'
+---
+
+{% include youtube.html id="tWAg1fIDNCg" %}
+
+{% include release_button.html release_tag="0.9.13" %}
+
+The __0.9.13 release of CARLA__ is finally here, so read on to see how the new improvements will revolutionize your workflow. 
+
+The most exciting new feature is the [__instance semantic segmentation__](#instance-semantic-segmentation)! A new camera sensor can be used to output images with the semantic tag and a unique ID for each world object embedded in the channels of the image file. Now the ground truth ID is available to differentiate objects of the same type in the camera field of view for training and evaluating networks to differentiate  and track overlapping objects of the same class. 
+
+[__Runtime texture streaming__](#runtime-texture-streaming) is now possible through the CARLA API. This new functionality enables the rendering material of objects in the scene to be updated at runtime without requiring the Unreal Editor. This can help produce training data to avoid overfitting pitfalls.
```

Comment:
> ... enables the rendering material of objects ...

Isn't this weirdly constructed?

Reply:
I think it is probably OK, I will think of a better way to phrase it, but will worry about other things for now